Content.Server/DeadSpace/MutilatedCorpse/MutilatedCorpseSystem.cs

```diff
@@ -1,4 +1,6 @@
+// Мёртвый Космос, Licensed under custom terms with restrictions on public hosting and commercial use, full text: https://raw.githubusercontent.com/dead-space-server/space-station-14-fobos/master/LICENSE.TXT
```

Comment:
```suggestion
// Мёртвый Космос, Licensed under custom terms with restrictions on public hosting and commercial use, full text: https://raw.githubusercontent.com/dead-space-server/space-station-14-fobos/master/LICENSE.TXT

```
Добавь пустую строку